blueocean-core-js/src/js/services/DefaultSSEHandler.js

```diff
@@ -137,7 +139,17 @@ export class DefaultSSEHandler {
      * @private
      */
     _updateRun(event, href) {
-        this.activityService.fetchActivity(href, { useCache: false }).then((run) => {
+        const pipelineHref = this._computePipelineHref(event);
+        const logMessage = `${event.jenkins_event} for pipeline ${pipelineHref} with run ${href}`;
+
+        if (!this.pipelineService.hasItem(pipelineHref)) {
+            this.loggingEnabled && console.log(`aborting fetch for ${logMessage}`);
+            return;
+        }
+
+        this.loggingEnabled && console.log(`fetch ${logMessage}`);
```

Comment:
why `console.log`?

Reply:
Still waiting for that new logging framework I keep hearing so much about ;) `loggingEnabled` is set to false so these won't actually be logging anything to console.